apps/foundation-demo/src/app/product/product.module.ts

```diff
@@ -23,7 +24,8 @@ import { BestSellersComponent } from './components/best-sellers/best-sellers.com
     MiscModule,
     EffectsModule.forFeature([
       ProductEffects
-    ])
+    ]),
+    ContainerModule
```

Comment:
I think this should be imported to `product.module` through the `design.module`.  Especially since the design folder is going to become a @daffodil/module of its own. Adding a single component from the @daffodil/design module would be like importing a specific component from angular/material, which would be kind of weird.

Reply:
Did it this way because @damienwebdev said we should only import where it's being used.

Reply:
I disagree, but ok

Reply:
@lderrickable can you elaborate on why you disagree?

Reply:
that makes sense